### PR TITLE
feat: reliable integrations [part 1]

### PR DESCRIPTION
**Pull request by andreabadesso**
### Acceptance Criteria
- We should remove all files from the old daemon

### Security Checklist
- [ ] Make sure you do not include new dependencies in the project unless strictly necessary and do not include dev-dependencies as production ones. More dependencies increase the possibility of one of them being hijacked and affecting us.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,60 +8,45 @@
...
-    "aws-sdk": "^2.878.0",
-    "axios": "^0.21.1",
+    "@hathor/wallet-lib": "^1.0.2",
+    "@types/jest": "^29.5.4",
+    "@types/ws": "^8.5.5",
+    "assert": "^2.1.0",
+    "aws-sdk": "^2.1454.0",
+    "axios": "^0.21.4",
     "dotenv": "^8.2.0",
+    "jest": "^29.6.4",
```
Comment (glevco) [inline package.json]: Shouldn't jest be a dev dependency?

**Review by glevco — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -8,60 +8,45 @@
     "src"
   ],
   "engines": {
-    "node": ">=10"
+    "node": ">=18"
```
Comment (r4mmer) [inline package.json:11]: We need to check if the lib works well with node 18.

File: package.json (hunk 1 above)
Comment (r4mmer) [reply package.json]: Same for the @types/jest

**Review by r4mmer — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (andreabadesso) [reply package.json]: Fixed in aca440380aea3b7eb464f02a9d2a3a10fd1b9eb2

File: package.json (hunk 2 above)
Comment (andreabadesso) [reply package.json:11]: I did a full sync using it, apparently it's working properly

**Pull request merged**